pkg/controller/dashboards/dashboard_controller.go

```diff
@@ -0,0 +1,160 @@
+package dashboards
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+
+	operv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/cluster-network-operator/pkg/apply"
+	cnoclient "github.com/openshift/cluster-network-operator/pkg/client"
+	"github.com/openshift/cluster-network-operator/pkg/controller/statusmanager"
+	"github.com/openshift/cluster-network-operator/pkg/names"
+	"github.com/openshift/cluster-network-operator/pkg/render"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	v1coreinformers "k8s.io/client-go/informers/core/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog/v2"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+const (
+	manifest = "bindata/dashboards/configmaps.yaml"
+)
+
+var (
+	//go:embed netstats.json
+	netstatsContent string
+	//go:embed ovn-health.json
+	ovnHealthContent string
+	dashboardRefs    []dashboardRef = []dashboardRef{
+		{
+			name:      "grafana-dashboard-network-stats",
+			json:      netstatsContent,
+			tplSuffix: "NetStats",
```

Comment:
super nit: feel free to ignore - put a comment here that if they alter `tplsuffixs`, they must update the config maps?